contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -47,74 +47,5 @@ You can also help with some existing issues under this category available at [de
 - Once the PR is accepted, close the branch.
 - After the PR is merged the development branch in the forked repository can be deleted.
 
-# Git Cheat Sheet 
-
-Fork the openebs/openebs repository into your account, referred in the below instructions as $user. 
-
-## Setting up Development Environment for OpenEBS projects
-```
-working_dir=$GOPATH/src/github.com/openebs
-mkdir -p $working_dir
-cd $working_dir
-```
-
-Set `user` to match your github profile name:
-
-```
-user={your github profile name}
-```
-
-## The steps mentioned here are w.r.t contribution towards openebs/openebs project
-```
-git clone https://github.com/$user/openebs.git
-cd openebs/
-git remote add upstream https://github.com/openebs/openebs.git
-git remote set-url --push upstream no_push
-git remote -v
-```
-
-## Synchronizing your local master(i.e. your $user/openebs project) with upstream master(i.e. openebs/openebs)
-```
-git checkout master
-git fetch upstream master
-git rebase upstream/master
-git status
-git push origin master
-```
-
-## Create Branch (based on feature or bug name)
-```
-git branch <branch_name>
-git checkout <branch_name>
-git push --set-upstream origin <branch_name>
-```
-
-## Synchronizing (i.e rebase) your local branch with upstream master
-```
-git checkout <branch-name>
-git fetch upstream master
-git rebase upstream/master
-git status
-git push
-```
-
-## Make changes to your local branch
-```
-# make your changes
-# keep fetching the commits from upstream master & rebase them here
-git commit
-git push
-
-# submit the PR to upstream from browser link https://github.com/$user/openebs
-```
-
-## Delete Branch
```

Comment:
Branch = branch

Reply:
That section has already been changed, a review has to be casted on the changed category. That's the right side.